recipes-devtools/clang/lldb_git.bb

```diff
@@ -45,6 +45,8 @@ EXTRA_OECMAKE="\
 
 EXTRA_OEMAKE = "VERBOSE=1"
 
+CXXFLAGS_append = " -Wno-error=format-security"
```

Comment:
I guess this is gcc specific, do please mark it with
```
CXXFLAGS_append_toolchain-gcc = ....
```

Reply:
Thanks, I have fixed this.